charts/index.yaml

```diff
@@ -9,7 +31,8 @@ entries:
       name: secrets-store-csi-driver
       repository: https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/master/charts
       version: 0.0.22
-    description: A Helm chart to install the Secrets Store CSI Driver and the Azure Keyvault Provider inside a Kubernetes cluster.
+    description: A Helm chart to install the Secrets Store CSI Driver and the Azure
+      Keyvault Provider inside a Kubernetes cluster.
```

Comment:
Is this line break intentional? I could see the same on other description fields too.

Reply:
The line breaks are generated by `helm` as part of `helm package` command.